tests/utils/test_geometry_comparisons.py

```diff
@@ -217,6 +219,24 @@ def test_compute_relative_rotation_angle(self):
 
         np.testing.assert_allclose(computed_deg, expected_deg, rtol=1e-3, atol=1e-3)
 
+    def test_compute_relative_rotation_angle_euler(self) -> None:
+        """"Sample 100 rotations about each individual Euler angle, and ensure Scipy and GTSAM result match."""
+        num_samples = 100
+        for axis in ["x", "y", "z"]:
+            for _ in range(num_samples):
+                theta = np.random.rand() * 360
+                R_1 = Rot3(Rotation.from_euler(axis, 0, degrees=True).as_matrix())
+                R_2 = Rot3(Rotation.from_euler(axis, theta, degrees=True).as_matrix())
```

Comment:
Why do we just test on the rotations with non-zero value on just one axis?

Reply:
just to make sure i understood your question -- you were wondering why i am generating a second rotation with only a single Euler angle?

Reply:
i was doing that just because it made the axis-angle computation simple.

Reply:
Do you think we miss out on some piece of logic when we have non-zero angle in just one axis?

I think we should add one more test which is very comprehensive (i.e. non-zero values in each axis for both R1 and R2)